src/main/java/io/github/crucible/grimoire/Grimoire.java

```diff
@@ -109,25 +129,30 @@ public void loadAllMixins() {
     private File extractResources(Collection<String> resources) throws IOException {
         File tmp = Files.createTempDirectory("grimoire").toFile();
         for (String resource : resources) {
-            try (InputStream resourceStream = getClass().getResourceAsStream('/'+resource)) {
+            try (InputStream resourceStream = this.getClass().getResourceAsStream('/'+resource)) {
                 if (resourceStream == null) {
                     logger.warn("[Grimoire] Failed to copy embedded patch with name {}", resource);
                     continue;
                 }
                 File extracted = new File(tmp, resource.replace('/','_'));
 
-                if (shouldLog)
+                if (this.shouldLog) {
                     logger.info("[Grimoire] Extracting embedded patch {} to {}", resource, extracted.getAbsolutePath());
+                }
                 Files.copy(resourceStream, extracted.toPath());
             }
         }
         return tmp;
     }
 
-    private boolean isMixinConfiguration(String name, boolean legacy) {
-        String splitName = ((name.contains("/")) ?
-                name.substring(name.lastIndexOf("/")).replace("/", "") :
-                name); // Caso o zip esteja usando o "\" esse zip está quebrado, já que o padrão é o "/".
+    private boolean isMixinConfiguration(String name, boolean legacy, boolean fullpath) {
```

Comment:
Is it necessary to add a full path check? The function itself will only split if the name has / which means it's a resource/file and will try to get the resource file name. It will also not support windows paths since it was designed to handle only zip file entries.

Reply:
Well, it's been in place before, so I left it be for instances where zip entries specifically are checked, passing false for normal files.